setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
     platformdirs>=2,<3
     importlib-metadata>=0.12;python_version<"3.8"
     importlib-resources>=1.0;python_version<"3.7"
-python_requires = >=3.6
```

Comment:
@asottile any idea why this seems to not like the existing restriction? 😆

Reply:
I think it's due to the `Python :: 2` classifiers maybe?  the logic for this is all kinda fiddly :S